R/pkg/R/DataFrame.R

```diff
@@ -1113,6 +1113,12 @@ setMethod("subset", signature(x = "DataFrame"),
             x[subset, select, ...]
           })
 
+setMethod("select", signature(x = "DataFrame"),
```

Comment:
Since 'select' is an external API, should we not put the code for the function after the roxygen codes?